modules/preview-react/pill/lib/PillLabel.tsx

```diff
@@ -1,36 +1,38 @@
 import React from 'react';
 
-import {createSubcomponent, styled, StyledType} from '@workday/canvas-kit-react/common';
-import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+import {createSubcomponent} from '@workday/canvas-kit-react/common';
+import {BoxProps, mergeStyles} from '@workday/canvas-kit-react/layout';
 import {OverflowTooltip, OverflowTooltipProps} from '@workday/canvas-kit-react/tooltip';
 
 import {usePillModel} from './usePillModel';
-import {colors} from '@workday/canvas-kit-react/tokens';
+
+import {createStencil} from '@workday/canvas-kit-styling';
 
 export interface PillLabelProps extends BoxProps {
   tooltipProps?: Omit<OverflowTooltipProps, 'children'>;
 }
-const StyledLabelContainer = styled(Box.as('span'))<StyledType>({
-  whiteSpace: 'nowrap',
-  overflow: 'hidden',
-  textOverflow: 'ellipsis',
-  display: 'block',
+
+export const pillLabelStencil = createStencil({
+  base: {
+    whiteSpace: 'nowrap',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    display: 'block',
```

Comment:
Do we want to keep this or import `ellipsisStyles`? 

Either way I'm good with.

Reply:
totally forgot we had that